build/README.md

```diff
@@ -15,3 +15,19 @@ docker build -f build/Dockerfile -t inklabs/rangedb:local .
 ```
 docker run -p 8080:8080 inklabs/rangedb:local
 ```
+
+## Using docker-compose to build & run locally in one step
+
+### RangeDB InMemory
+* From root dir
+    * docker-compose -f build/docker-compose.yml up --build
+* If inside build directory
+    * docker-compose up --build
```

Comment:
Stick with one strategy. I like the first: from root dir.

Reply:
removed!